modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/entitlement.properties

```diff
@@ -0,0 +1,71 @@
+PAP.Enable={{identity.entitlement.policy_point.pap.enabled}}
+PDP.DecisionCaching.Enable={{identity.entitlement.policy_point.pdp.caching.decision_caching.enabled}}
+#cache intervals are in seconds
+PDP.DecisionCaching.CachingInterval={{identity.entitlement.policy_point.pdp.caching.decision_caching.caching_interval}}
+PDP.AttributeCaching.Enable={{identity.entitlement.policy_point.pdp.caching.attribute_caching.enabled}} 
+PDP.AttributeCaching.CachingInterval={{identity.entitlement.policy_point.pdp.caching.attribute_caching.caching_interval}}
+PDP.ResourceCaching.Enable={{identity.entitlement.policy_point.pdp.caching.resource_caching.enabled}} 
+PDP.ResourceCaching.CachingInterval={{identity.entitlement.policy_point.pdp.caching.resource_caching.caching_interval}}
+PDP.SchemaValidation.Enable={{identity.entitlement.policy_point.pdp.schema_validation_enabled}}
+PDP.Balana.Config.Enable={{identity.entitlement.policy_point.pdp.balana_config_enabled}}
+PDP.Multiple.Decision.Profile.Enable={{identity.entitlement.policy_point.pdp.multiple_decision_profile_enabled}}
+PDP.Global.Policy.Combining.Algorithm={{identity.entitlement.policy_point.pdp.global_policy_combining_algorithm}}
+PAP.Policy.Add.Start.Enable={{identity.entitlement.policy_point.pap.policy_add_start_enable}}
+#PAP.Policy.Add.Start.Policy.File.Path=
+PAP.Items.Per.Page={{identity.entitlement.policy_point.pap.items_per_page}}
+PDP.Registry.Level.Policy.Cache.Clear={{identity.entitlement.policy_point.pdp.registry_level_policy_cache_clear}}
+PDP.PolicyCaching.CachingInterval={{identity.entitlement.policy_point.pdp.caching.policy_caching.caching_interval}}
+Entitlement.Engine.CachingInterval={{identity.entitlement.entitlement_engine_caching_interval}}
+PDP.References.MaxPolicyEntries={{identity.entitlement.policy_point.pdp.reference_max_policy_entries}}
+
+#Enable JSON shorten form support by default
+JSON.Shorten.Form.Enabled={{identity.entitlement.JSON_shorten_form_enabled}}
+
+
+#PDP.Extensions.Extension.1=your.extension.class.name
+
+
+#PDP.Policy.Store=org.wso2.carbon.identity.entitlement.policy.store.CarbonRegistryPolicyStore
+
+PIP.AttributeDesignators.Designator.1={{identity.entitlement.policy_point.pip.attribute_designators.designator_1}}
+PIP.AttributeDesignators.Designator.2={{identity.entitlement.policy_point.pip.attribute_designators.designator_2}}
+PIP.ResourceFinders.Finder.1={{identity.entitlement.policy_point.pip.resource_finder}}
+
+PAP.Entitlement.Data.Finder.1={{identity.entitlement.policy_point.pap.entitlement_data_Finder}}
+PAP.Policy.Publisher.Module.1={{identity.entitlement.policy_point.pap.policy_publisher_module}}
+#PAP.Policy.Post.Publisher.Module.1=
+#PAP.Policy.Publisher.Verification.Handler=
+PAP.Policy.Version.Module={{identity.entitlement.policy_point.pap.policy_version_module}}
+PAP.Status.Data.Handler.1={{identity.entitlement.policy_point.pap.status_data_handler}}
+
+PDP.Policy.Finder.1={{identity.entitlement.policy_point.pdp.policy_finder}}
```

Comment:
Use loops for the places which can have multiple items.

Reply:
Fixed.